app/uk/gov/hmrc/tariffclassificationfrontend/views/partials/case_nav.scala.html

```diff
@@ -49,6 +49,12 @@
         title = messages("case.menu.activity"),
         call = uk.gov.hmrc.tariffclassificationfrontend.controllers.routes.CaseController.activityDetails(reference)
     )
+
+    @case_nav_item(
+        active = page == CaseDetailPage.KEYWORDS,
+        title = messages("case.menu.keywords"),
+        call = uk.gov.hmrc.tariffclassificationfrontend.controllers.routes.CaseController.keywordsDetails(reference)
```

Comment:
you could import `uk.gov.hmrc.tariffclassificationfrontend.controllers.routes.CaseController` above
Then the calls would be simple to read

Reply:
Done and fixed all the other ones in this file ;)